chia/cmds/plotters.py

```diff
@@ -0,0 +1,9 @@
+import click
+from chia.plotters.plotters import call_plotters
+
+
+@click.command("plotters", context_settings={"ignore_unknown_options": True}, add_help_option=False)
```

Comment:
Can we use `@click.group("plotters", short_help="Advanced plotting options")` (or some other helpful text)? Running `chia -h` shows the plotters command, but doesn't have any help text next to is

Reply:
`Usage: chia [OPTIONS] COMMAND [ARGS]...

  Manage chia blockchain infrastructure (1.2.6.dev40)

Options:
  --root-path PATH  Config file root  [default: /Users/jeffc/.chia/mainnet]
  -h, --help        Show this message and exit.

Commands:
  configure   Modify configuration
  farm        Manage your farm
  init        Create or migrate the configuration
  keys        Manage your keys
  netspace    Estimate total farmed space on the network
  plotnft     Manage your plot NFTs
  plots       Manage your plots
  plotters
  run_daemon  Runs chia daemon
  show        Show node information
  start       Start service groups
  stop        Stop services
  version     Show chia version
  wallet      Manage your wallet`